sched/mqueue/mq_rcvinternal.c

```diff
@@ -69,17 +68,27 @@
  *
  ****************************************************************************/
 
-int nxmq_verify_receive(FAR struct mqueue_inode_s *msgq,
-                        int oflags, FAR char *msg, size_t msglen)
+#ifdef CONFIG_DEBUG_FEATURES
+int nxmq_verify_receive(FAR struct file *mq, FAR char *msg, size_t msglen)
```

Comment:
Initially I was thinking you forgot to declare this function when no DEBUG FEATURES as enabled, but I saw you did it at sched/mqueue/mqueue.h ! Very nice! ostest passed correctly!